### PR TITLE
Initial Changes to README

### PR DESCRIPTION
**Pull request by tapankavasthi**
- A bit of formatting here, and there
- A few links in and around
- Gave it a few SEO coins
- Gave it a poetic conclusion

Kudos, Sherub! Good to know about this lecture series.

[Preview Changes](https://github.com/tapankavasthi/learning-haskell)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,75 @@
...
+
+HTTP/2 302
+server: nginx/1.17.8
+date: Mon, 20 Apr 2020 15:55:07 GMT
+content-type: text/html
+content-length: 145
+location: https://raw.githubusercontent.com/commercialhaskell/stack/stable/etc/scripts/get-stack.sh
+```
+
+With the header information available to us, We can now see the actual location of our script.
```
Comment (sherubthakur) [inline README.md:33]: Can we shift this part into FAQ. I would like the readme to be like a no-brainer thing first and foremost. And if folks are interested in knowing more let's address this using an FAQ section.

File: README.md [hunk 2]
```diff
@@ -1,19 +1,75 @@
...
+Leaving GHCi.
+```
+
+
+## Project Setup
+We can start with the usual [forking and cloning](https://help.github.com/en/github/getting-started-with-github/fork-a-repo)  of this repository. After that, we should be able to build and test our project using _stack_:
+
+```shell script
+$ cd learning-haskell
+$ stack build --test
```
Comment (sherubthakur) [inline README.md:64]: Opinion: `stack test` should do everything for you and is less verbose.

**Review by nixypanda — COMMENTED** [review]
This looks great man. I will merge this into the main branch as soon as we move the flag section to FAQ section. Thanks!

File: README.md (hunk 2 above)
Comment (tapankavasthi) [reply README.md]: Yes, both are synonyms - yes?
I've changed it to `stack test` as per your suggestion.

File: README.md (hunk 1 above)
Comment (tapankavasthi) [reply README.md]: @sherubthakur  Done 🙂

**Comment by tapankavasthi** [comment]
> This looks great man. I will merge this into the main branch as soon as we move the flag section to FAQ section. Thanks!

Thanks, @sherubthakur for taking a look. I've addressed your review comments. Please take a second look at your convenience.
Btw, the idea of including a FAQ guide was spot on!

**Review by nixypanda — APPROVED** [review]

**Pull request merged**